project/styles/components/content/facts-timetable-regular.scss

```
@@ -0,0 +1,40 @@
+.facts-timetable-regular {
+  border-left: 2px solid $grey2;
+  padding: 12px 24px;
+  max-width: 400px;
+  margin: 24px;
+}
+
+.fact-timetable-regular__title {
+  font-family: $bold;
```

graag bestaande title mixins gebruiken en responsive kloppend maken.

```suggestion
@include title3;
```